recipes/nordugrid-arc/meta.yaml

```diff
@@ -0,0 +1,97 @@
+{% set name = "nordugrid-arc" %}
+{% set version = "6.6.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://source.coderefinery.org/nordugrid/arc/-/archive/v{{ version }}/arc-v{{ version }}.tar.gz
+  sha256: 898482005ebd80221bef920ff175987a286d250212dfcf3855523358377eefc3
+
+build:
+  number: 0
+  skip: true  # [win or osx]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - pkg-config
+    - make
+    - autoconf
+    - automake
+    - libtool
+    - swig
```

Comment:
swig should be a host tool

Reply:
That's good to know, thanks and fixed.

@conda-forge/staged-recipes ready for another review